lib/ReactViews/Map/Panels/SharePanel/Print/PrintWorkbench.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import Workbench from "../../../../../Models/Workbench";
+
+import CatalogMemeberMixin from "../../../../../ModelMixins/CatalogMemberMixin";
+import MappableMixin from "../../../../../ModelMixins/MappableMixin";
+import { BaseModel } from "../../../../../Models/Definition/Model";
+import DiscretelyTimeVaryingMixin from "../../../../../ModelMixins/DiscretelyTimeVaryingMixin";
+import Legend from "../../../../Workbench/Controls/Legend";
+import SelectableDimensions, {
+  DEFAULT_PLACEMENT,
+  filterSelectableDimensions,
+  findSelectedValueName
+} from "../../../../../Models/SelectableDimensions";
+import { useTranslation } from "react-i18next";
+
+interface Props {
+  workbench: Workbench;
+}
+
+const renderDisplayVariables = (catalogItem: BaseModel & any) => {
```

Comment:
Can we get rid of the `any` here?

```suggestion
const renderDisplayVariables = (catalogItem: BaseModel) => {
```

Reply:
yup, will do